crates/arrow_tools/src/format.rs

```diff
@@ -327,6 +328,64 @@ fn column_indices(batch: &RecordBatch, column_names: &[String]) -> Result<Vec<us
         .collect()
 }
 
+fn hashmap_to_string(map: &HashMap<String, String>, separator: &str) -> String {
+    let mut keys: Vec<_> = map.keys().collect();
+    keys.sort(); // To make this function reproducible
+    keys.iter()
+        .map(|k| format!("* {}: {}", k, map.get(*k).map_or("", String::as_str)))
```

Comment:
I think this should be okay because you know the keys are present in the map.
```suggestion
        .map(|k| format!("* {}: {}", k, map[k]))
```